pyproject.toml

```diff
@@ -41,7 +41,7 @@ ignore-module = false
 ignore-nested-functions = false
 ignore-nested-classes = true
 ignore-setters = false
-fail-under = 61.0
+fail-under = 62.5
```

Comment:
Why was this bumped up slightly?

Reply:
To increase the minimum docstring coverage required ( so we keep on improving our docsting coverage).

Reply:
Ok nice, thanks 🙂